components/BottomNavigation/src/ColorThemer/MDCBottomNavigationBarColorThemer.h

```diff
@@ -0,0 +1,35 @@
+/*
+ Copyright 2017-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import "MaterialThemes.h"
+#import "MaterialBottomNavigation.h"
+
+/**
+ Used to apply a color scheme to theme MDCBottomNavigationBar.
```

Comment:
I think it would be good to provide a description of what the themer will do (and what it won't do). For example, does it apply `tintColor`, `backgroundColor`, selected item color, etc.?  Are there any colors that aren't changed/updated that clients should expect to change manually?

Reply:
I've elaborated on how this affects the various color properties of the bottom navigation bar. The client shouldn't have to change any colors manually when using the themer.